src/auth/providers.js

```diff
@@ -27,7 +27,10 @@ function getDeviceInfo(deviceId, appId, appVersion = '') {
 
 function anonProvider(auth) {
   return {
-    authenticate: data => {
+    /**
+     * Login to a stitch application using anonymous authentication
+     */
```

Comment:
We should als specify that this returns a promise using `@returns` - similar to what's in client.js L67.